src/v/metrics/metrics.h

```diff
@@ -24,6 +24,11 @@
 
 namespace metrics {
 
+// Common labels used for metrics;
+const ss::metrics::label namespace_label{"namespace"};
```

Comment:
declare globals in the header, but define them in the .cc.

Reply:
Nice catch, will do.